macros/columns/base/string.sql

```diff
@@ -30,96 +39,102 @@
     {{ dbt_synth_data.synth_store('final_fields', name, final_field) }}
 {%- endmacro %}
 
-{# {% macro sqlite__synth_column_string_base(name, min_length, max_length) %}
```

Comment:
Why delete commented sqlite and duckdb code?

Reply:
I had in mind to remove what we don't use.
We will do a lot of improvements, but we'll not spend time with sqlite and duckdb.
But it is an open source stuff, and it will be good to keep it.
Ok. I will put it back.

Reply:
reverted all deleted/changed code - duckdb, sqlite, postgres